lib/components/popups/course-search/Cart.tsx

```diff
@@ -169,64 +169,17 @@ const Cart: FC<{ allCourses: SISRetrievedCourse[] }> = (props) => {
     extras: SearchExtras,
   ): Promise<[SISRetrievedCourse[], number[]]> => {
     return new Promise(async (resolve) => {
-      // let courses: SISRetrievedCourse[] = [...courseCache]; // how actively is this cache updated?
-      // if (!retrievedAll) { // how often is this filter used?
       const retrieved: any = await axios
-        .get(getAPI(window) + '/search', {
+        .get(getAPI(window) + '/cartSearch', {
           params: getParams(extras),
         })
         .catch(() => {
           return [[], []];
         });
-      let SISRetrieved: SISRetrievedCourse[] = processedRetrievedData(
-        retrieved.data.data,
-        extras,
-      );
-      return resolve([SISRetrieved, []]);
-      // } else {
-      //   const filterProcessing = filterCourses(extras, courses);
-      //   if (filterProcessing) return filterProcessing;
-      //   return resolve([courses, []]);
-      // }
```

Comment:
no need to process retrieved data because both `/search` and `/cartSearch` routes are more precise. 

`200      // TODO: backend searches for courses with "None" area. Fix this.
`
this todo is done.

Reply:
see 
<img width="636" alt="Screenshot 2022-12-28 at 9 23 38 PM" src="https://user-images.githubusercontent.com/90944895/209895094-cf234a71-4bc8-4253-853f-45ed913cbe80.png">